modules/lightning_telemetry/CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+Failures to send Lightning Telemetry are intended to be graceful and quiet. Exceptions are caught and output is intentionally suppressed.
+
+To disable error suppression, set the environmental variable LIGHTNING_TELEMETRY_LOUD to a value that evaluates as true.
+E.g., `LIGHTNING_TELEMETRY_LOUD=1 drush cron`.
```

Comment:
Ideally this would be a configuration value, not an environment variable.

Reply:
Eh. Like a real CM one?

Reply:
I would suggest the State system. Environment variables aren't durable or convenient enough, and configuration appears in configuration exports and travels between environments, which you don't want.

Reply:
Also, why is this in `CONTRIBUTING.md`? It seems more like README material.

Reply:
Because it's really only for people developing this module. I would never expect an end user to use this variable.

Reply:
I don't know if we need a module-specific CONTRIBUTING.md. I'm +1 for @TravisCarden's idea of using a state setting to disable error suppression.

Reply:
As a developer, how would I quickly set the state from the CLI?

Reply:
Ah, `drush sset`.